.editorconfig

```diff
@@ -35,6 +35,11 @@ dotnet_diagnostic.IDE0005.severity = error
 # Yell at people when they don't include the file header template
 dotnet_diagnostic.IDE0073.severity = error
 
+# shut IDEs up about file scoped namespaces and java-style using directives 
+# while there is no authoritative style rule found in here
+dotnet_diagnostic.IDE0065.severity = none
+dotnet_diagnostic.IDE0161.severity = none
```

Comment:
`csharp_using_directive_placement = outside_namespace:error`

Reply:
good point :P
i honestly did not pay attention to that

Reply:
Not well known, no worries

Reply:
removed that one